src/App.js

```diff
@@ -1,77 +1,33 @@
-import mockup from './Images/mockup.png';
-import sunIcon from './Images/icon-sun.png';
-import './App.css';
-import Button from './Components/Button';
-import CourseTopicCard from './Components/CourseTopicCard';
-import './Components/styles.css';
+import mockup from "./Images/mockup.png";
+import sunIcon from "./Images/icon-sun.png";
+import "./App.css";
+import Button from "./Components/Button";
+import CourseTopicCard from "./Components/CourseTopicCard";
+import "./Components/styles.css";
+import NavigationBar from "./Components/NavigationBar";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import Apply from "./Pages/Apply";
+import Apps from "./Pages/Apps";
+import Courses from "./Pages/Courses";
+import Home from "./Pages/Home";
+import Support from "./Pages/Support";
+import Team from "./Pages/Team";
```

Comment:
we should probably start organizing our imports - how does splitting imports into two sections (each alphabetized) sound? 

section 1: third party code (`react-router-dom`)
section 2: local imports (everything else like `./Pages`

Reply:
Great idea! Not organizing the imports is probably fine when the number of imports is small, but once it gets large, we definitely should be doing that.